frontend/packages/app/src/clutch.config.js

```diff
@@ -46,4 +46,41 @@ module.exports = {
       },
     },
   },
+  "@clutch-sh/experimentation": {
+    listExperiments: {
+      trending: true,
+      componentProps: {
+        columns: ["identifier", "targets", "type", "description"],
+        experimentTypes: {
+          "type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestSpecification": {
+            mapping: e => {
+              const config = e.latency || e.abort;
+              const clusters = config.clusterPair;
+              return {
+                targets: `from "${clusters.downstreamCluster}" to "${clusters.upstreamCluster}"`,
```

Comment:
Nit: I'd make this less verbose, e.g.,: `"${clusters.downstreamCluster}" <right arrow icon> "${clusters.upstreamCluster}"`

Reply:
👍